qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,107 @@
+It helps us optimize performance in "real-world" situations.
+
+For our v1.7.0 release we introduced sparse vectors. As noted in our [Qdrant v.1.7.0 release announcement](https://qdrant.tech/articles/qdrant-1.7.x/#sparse-vectors), we store sparse vectors differently and keep the non-zero dimensions.
+
+As an additional improvement, we used the [NeurIPS 2023 datasets](https://big-ann-benchmarks.com/neurips23.html) which are a standard to evaluate vector search algorithms.
+
+We optimized our search to improve throughput by a factor of 16.
+
+This is based on our test benchmarks in this [Sparse vectors benchmark](https://github.com/qdrant/sparse-vectors-benchmark) repository. For those benchmarks, we ran
+the full query dataset againt a system with 8 CPUs.
+
+Based on our tests, you can expect significant performance improvements. 
+The performance that we see is shown on the following two-dimensional histogram:
+
+![Histogram with increasing latency for higher query dimensions](/blog/qdrant-1.8.x/neurIPS_bench_example.png)
+
+The colors within the scatter plot show the frequency of the results. The "red"
+points show the highest frequency.
+
+## Optimized CPU use
+
+We continue to optimize our search. With dynamic CPU saturation we've increased
+indexing speed, with no impact on search requests.
+
+This version introduces an `optimizer_cpu_budget` parameter to control the maximum number of CPUs used
+for indexing. For most users, the default `optimizer_cpu_budget` works well. It allocates more CPU resources
+for building indexes faster, while reserving resources to handle searches.
+
+![Allocation of CPU resources](/blog/qdrant-1.8.x/cpu-management.png)
+
+With our [Collections](/documentation/concepts/collections/) API, you can 
+configure how Qdrant saturates the CPUs in your configuration. 
+
+## Optimize RAM with immutable text fields
+
+We have optimized the required RAM with immutable text fields. We have set up
+a field index which helps minimize what is stored.
+
+Based on our tests of a system with 64GB of RAM, we've reduced by the amount of 
+required RAM by around 10%. We expect greater improvements on systems with less RAM.
+
+Mutable documents require additional RAM.
+
+## Increase search performance
+
+To improve search performance we have optimized the way we load documents for searches with a text field index. This also minimizes the load on RAM. 
+
+To set this up, we load documents mostly sequentially, in increasing order.
+
+## Minor improvements and new features
+
+Beyond these enhancements, Qdrant v1.8.0 adds and improves several features:
+
+<!-- Requires merging https://github.com/qdrant/landing_page/pull/631 -->
+1. Order results by payload value with the Scroll API. For more information,
+   see how you can [Order points by a payload key](/documentation/concepts/points/#order-points-by-a-payload-key).
+1. [Implement datetime support for the payload index](https://github.com/qdrant/qdrant/issues/3320)
```

Comment:
Maybe you want to link to the documentation page (after merging #678) instead of linking the issue.

Reply:
When we release v1.8 (and as @xzfc suggests, after merging #678) 

```suggestion
1. [Implement datetime support for the payload index](/documentation/concepts/filtering/#datetime-range)
```